packages/cli/src/commands/account/balance.ts

```diff
@@ -23,15 +25,27 @@ export default class Balance extends BaseCommand {
   ]
 
   async run() {
-    const kit = await this.getKit()
+    const client = await this.getPublicClient()
     const { args, flags } = await this.parse(Balance)
 
-    console.log('All balances expressed in units of 10^-18.')
-    printValueMap(await kit.getTotalBalance(args.arg1 as string))
+    console.log('All balances expressed in units of wei.')
```

Comment:
nit can we use the oclif ux logging tool here?

Reply:
decided to leave `console.log`